packages/compass-components/package.json

```diff
@@ -22,12 +22,13 @@
     "compile-ts": "tsc -p tsconfig.json",
     "cover": "nyc npm run test",
     "depcheck": "depcheck",
-    "lint": "eslint src/**/*.tsx src/**/*.ts",
+    "lint": "eslint src/**/*.tsx src/**/*.ts && prettier --check .",
```

Comment:
Just noticed this here.. why eslint (and possibly also prettier) is targeting src only and not everything except the (dist / lib)? I don't see a reason to exclude other files unless we are ok with them be messed up